app/src/main/res/values-pl/strings.xml

```diff
@@ -95,4 +95,9 @@
   <string name="pref_wifi_only">Pobierz zawartość tylko poprzez WiFi</string>
   <string name="pref_autonightmode_summary">Przełączaj automatycznie między trybami dnia i nocy.</string>
   <string name="pref_autonightmode">Zautomatyzowany tryb nocny.</string>
+  <string name="pref_external_link_popup_title">Ostrzegaj przy zewnętrznych linkach</string>
```

Comment:
Localised strings are generated from https://en.wikipedia.org/wiki/Translatewiki.net and as such only the English version should be included as this will simply get overwritten.

Reply:
Good to know that